esmvalcore/cmor/_fixes/shared.py

```diff
@@ -29,6 +29,21 @@ def _get_altitude_to_pressure_func():
 altitude_to_pressure = _get_altitude_to_pressure_func()  # noqa
 
 
+def _get_pressure_to_altitude_func():
+    """Get function converting air pressure [Pa] to altitude [m]."""
+    base_dir = os.path.dirname(os.path.abspath(__file__))
+    source_file = os.path.join(base_dir, 'us_standard_atmosphere.csv')
+    data_frame = pd.read_csv(source_file, comment='#')
+    func = interp1d(data_frame['Pressure [Pa]'],
+                    data_frame['Altitude [m]'],
+                    kind='cubic',
+                    fill_value='extrapolate')
+    return func
+
+
+pressure_to_altitude = _get_pressure_to_altitude_func()  # noqa
```

Comment:
You could also use the standard library decorator [`lru_cache`](https://docs.python.org/3/library/functools.html#functools.lru_cache) to cache the the result. That would have the advantage that the module loads faster because the table only needs to be read when the function is called.

Reply:
So I just have to add `@lru_cache` to `_get_pressure_to_altitude_func` and then use `_get_pressure_to_altitude_func()` instead of `pressure_to_altitude`?